docs/content/how-to/openstack/additional-ports.md

```diff
@@ -0,0 +1,50 @@
+# Additional ports for Nodepools
+
+This document describes how to attach additional ports to nodes in a HostedCluster on OpenStack.
+
+## Use-cases
+
+- **SR-IOV**: Single Root I/O Virtualization (SR-IOV) is a specification that allows a single Peripheral Component Interconnect Express (PCIe) physical device to appear as multiple separate physical devices. This can be useful for high-performance networking scenarios. The Nodepool can be configured to use SR-IOV by attaching additional ports to the nodes. Workloads that require high-performance networking can use the SR-IOV devices.
+- **DPDK**: Data Plane Development Kit (DPDK) is a set of libraries and drivers for fast packet processing. DPDK can be used to improve the performance of network applications. The Nodepool can be configured to use DPDK by attaching additional ports to the nodes. Workloads that require high-performance networking can use the DPDK devices.
+- **Manila RWX on NFS**: Manila ReadWriteMany (RWX) volumes can be used by multiple nodes in a HostedCluster. The Nodepool can be configured to use Manila RWX volumes by attaching additional ports to the nodes. Workloads that require shared storage will be able to reach the NFS network configured for the Manila RWX volumes.
+- **Multus CNI**: Multus is a meta-plugin for Kubernetes CNI that enables attaching multiple network interfaces to pods. The Nodepool can be configured to use Multus by attaching additional ports to the nodes. Workloads that require multiple network interfaces will be able to use the Multus CNI. For example it's useful if the workload needs to connect to IPv6 networks (dual-stack or single-stack).
+
+## Prerequisite
+
+Additional networks must be created in OpenStack and the project used by the HostedCluster must have access to these networks.
+
+## Available options
+
+The `--openstack-node-additional-port` flag can be used to attach additional ports to nodes in a HostedCluster on OpenStack. The flag takes a list of parameters separated by commas. The parameter can be used multiple times to attach multiple additional ports to the nodes.
+
+The parameters are:
+
+| Parameter       | Description                                                                                          | Required | Default  |
+|-----------------|------------------------------------------------------------------------------------------------------|----------|----------|
+| `network-id`    | The ID of the network to attach to the node.                                                         | Yes      | N/A      |
+| `vnic-type`     | The VNIC type to use for the port. When not specified, Neutron uses the default `normal`.            | No       | N/A |
+| `port-security` | Whether to enable port security for the port.                                                        | No       | `true`   |
```

Comment:
nit: you say the default for `vnic_type` is `N/A`, presumably because we don't set it by default and rely on neutron's default, but then you specify a default here even though we're also relying on neutron's default. Should the default here be `N/A` also?

Reply:
right